Document per-container termination message limits (#33320)
kubernetes/website: content/en/docs/tasks/debug/debug-application/determine-reason-pod-failure.md

populate the Container's status message on both success and failure.

The termination message is intended to be brief final status, such as an assertion failure message.
The kubelet truncates messages that are longer than 4096 bytes.

> The total message length across all containers will be limited to 12KiB, divided equally among each container.

Reviewer: ```suggestion
The total message length across all containers is limited to 12KiB, divided equally among each container.
```

populate the Container's status message on both success and failure.

The termination message is intended to be brief final status, such as an assertion failure message.
The kubelet truncates messages that are longer than 4096 bytes.

The total message length across all containers will be limited to 12KiB, divided equally among each container.

> This means that if there are 12 containers (`initContainers` or `containers`), each has 1024 bytes of available termination message space.

Reviewer 2: ```suggestion
For example, if there are 12 containers (`initContainers` or `containers`), each has 1024 bytes of available termination message space.
```

> If there are 24 containers, each has 512 bytes, and so on.

Reviewer 2: I think this example is redundant because there's no new information being shared.
